### PR TITLE
Fix the broken link in Markdown

### PR DESCRIPTION
**Pull request by superbrothers**
**What this PR does / why we need it**: Fix the broken link in Markdown

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/openapi-spec/README.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ For example:
 ### `x-kubernetes-patch-strategy` and `x-kubernetes-patch-merge-key`
 
 Some of the definitions may have these extensions. For more information about PatchStrategy and PatchMergeKey see
-[strategic-merge-patch] (https://github.com/kubernetes/community/blob/master/contributors/devel/strategic-merge-patch.md).
+[strategic-merge-patch](https://github.com/kubernetes/community/blob/master/contributors/devel/strategic-merge-patch.md).
```
Comment (cblecker) [inline api/openapi-spec/README.md]: Can you also change this link to https://git.k8s.io/community/contributors/devel/strategic-merge-patch.md while you're at it? Thank you!

Comment (superbrothers) [reply api/openapi-spec/README.md]: I changed this link at 6b40582.

**Comment by cblecker** [comment]
@liggitt If you have a sec, would you mind approving this link fix?

**Pull request merged**